.ci/Jenkinsfile

```diff
@@ -86,6 +86,32 @@ pipeline {
             }
           }
         }
+        stage('End-To-End Tests') {
+          parallel {
+            options { skipDefaultCheckout() }
+            environment {
+              GO111MODULE = 'on'
+              PATH = "${env.WORKSPACE}/${env.BASE_DIR}/bin:${env.GOPATH}/bin:${HOME}/go/bin:${env.PATH}"
+              GOPROXY = 'https://proxy.golang.org'
+              STACK_VERSION = "7.6.0"
```

Comment:
Hardcoded. As a follow-up we could define a place for runtime dependencies (like a properties file or .env file)